src/display/annotation_layer.js

```diff
@@ -345,21 +345,21 @@ class AnnotationElement {
         const hidden = event.detail.display % 2 === 1;
```

Comment:
Nit: We could avoid some repetition in this function by doing the following.
```suggestion
        const { hidden } = event.detail;
        const hidden = display % 2 === 1;
```